### PR TITLE
Bh 2033 add fade in and fade out for relaxation

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
The relaxation sounds have a "fade in" and "fade out" setting which will result in a smoother start and end of the entire relaxation.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        auto setValue(bool value) -> void override;
+        auto getValue() const -> bool override;
+    };
+
+    class RelaxationFadeModel : public AbstractRelaxationFadeModel
+    {
+      public:
+        RelaxationFadeModel() = delete;
+
+        RelaxationFadeModel(ApplicationCommon *app)
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp:26]:
```suggestion
        explicit RelaxationFadeModel(ApplicationCommon *app) : fade{std::make_unique<RelaxationFadeSettings>(app)} {}
```

File: products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            fade = std::make_unique<RelaxationFadeSettings>(app);
+        }
+
+        gui::AbstractSettingsModel<bool> &getFade() override
```
Comment (Lefucjusz) [inline products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp:31]:
```suggestion
        auto getFade() -> gui::AbstractSettingsModel<bool> & override
```

To be consistent with the rest of this file :wink:

**Review by Lefucjusz — APPROVED** [review]

File: products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp (hunk 1 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp]: done

File: products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp (hunk 2 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/apps/common/include/common/models/RelaxationFadeModel.hpp]: done

**Pull request merged**